src/engraving/compat/midi/event.h

```diff
@@ -102,8 +102,8 @@ class NPlayEvent : public PlayEvent
 
     const Note* _note = nullptr;
     const Harmony* _harmony = nullptr;
-    int _origin = -1;
-    int _discard = 0;
+    size_t _origin = (size_t)-1;
```

Comment:
(size_t)-1 is safe, but it's rather std::numeric_limits<size_t>::max()
but then we have 'getOriginatingStaff() + 1' and it's confusing, even though will give us zero
maybe leave int everywhere?

Reply:
It was `int` everywhere. And lead to a warning, I beliebe it is this'
```
2023-05-09T13:31:26.9125539Z D:/a/MuseScore/MuseScore/src/importexport/midi/internal/midiexport/exportmidi.cpp(317): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
```

Reply:
But we can fix it with a single cast in exportmidi.cpp
I'll do that